src/pages/2020/join/index.js

```diff
@@ -340,346 +310,268 @@ const Wrapper = s.div`
+        
+        <Business background={businessImage}>
+        <section class="business">
         <h2>Business</h2>
-        <div class="section">
-          <p class="overall">
-            The Daily Pennsylvanian is a $3-million, student-run media organization.
-            Our incredible teams work together to market our products and sustain our business.
-            Through the DP, students experience working for a real company, with real staffers, clients, and revenues.
-            Join one of our teams to gain an invaluable, hands-on experience unlike any other.
-          </p>
-          <div class="departments">
-            <span><h4 class="biz-tab active" id="marketing">Marketing</h4></span>
-            <span><h4 class="biz-tab" id="consulting">Consulting</h4></span>
-            <span><h4 class="biz-tab" id="analytics">Analytics</h4></span>
-            <span><h4 class="biz-tab" id="product-lab">Product Lab</h4></span>
-            {/* <span><h4 class="biz-tab" id="circulation">Circulation</h4></span> */}
+          <div class="section">
+            <p class="overall">
+              The Daily Pennsylvanian is a $3-million, student-run media organization.
+              Our incredible teams work together to market our products and sustain our business.
+              Through the DP, students experience working for a real company, with real staffers, clients, and revenues.
+              Join one of our teams to gain an invaluable, hands-on experience unlike any other.
+            </p>
+
+            <div class="departments">
+              {JoinBusinessJSON.map((dpt, idx) => (
+              <span>
+                <h4 class={(businessActive == dpt.id) ? 'biz-tab active' : 'biz-tab'} id={dpt.id} onClick={() => {
+                  setBusinessImage(dpt.background)
+                  setBusinessDescription(dpt.text)
+                  setBusinessHighlights(dpt.highlights)
+                  setBusinessHighlightLinks(dpt["highlight-links"])
+                  setBusinessActive(dpt.id)
+                }}>
+                  {dpt.department}
+                </h4>
+              </span>
+              ))}
+            </div>
+            <div className="biz-text active">
+                {businessDescription.map(p => (
+                  <p>{p}</p>
+                ))}
+                <h5>{(businessHighlights.length == 0) ? '': 'Highlights'}</h5>
+                {
+                  (businessHighlights.length == businessHighlightLinks.length)
+                  ? businessHighlights.map((l, idx) => (
+                    <a href={businessHighlightLinks[idx]} target="_blank" rel="noreferrer">{l} &#8594;</a>
+                  ))
+                  : businessHighlights.map((l, idx) => (
+                    <p>{l}</p>
+                  ))
+                }
+            </div>
           </div>
-          <div class="biz-text active" id="marketing-text">
+        </section>
+        </Business>
+
+        <section class="street">
```

Comment:
Should make JSON files for both 34th street and under the button as well such that you are loading from the json files